mobile/android/app/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.10.2)
+project("native_buffer")
+
+add_library(native_buffer SHARED
+    src/main/cpp/native_buffer.c)
+
+find_library(log-lib log)
+
+target_link_libraries(native_buffer ${log-lib})
```

```suggestion
cmake_minimum_required(VERSION 3.12)

set(CMAKE_C_STANDARD 17)
set(CMAKE_C_STANDARD_REQUIRED ON)

project(native_buffer LANGUAGES C)

add_library(native_buffer SHARED
    src/main/cpp/native_buffer.c
)
```
I suggest to overwrite the whole compile script. It is so bad actually, no certain standard of C language, no limited languages for the whole project, no certain compiler, and import useless dynamic linking libraries, etc.